internal/token.go

```diff
@@ -1602,6 +1602,7 @@ expression_loop:
 				z.tt = TextToken
 				return z.tt
 			}
+			z.openBraceIsExpressionStart = true
```

Comment:
Not 100% sure this is in the right place but all tests seem to be passing 🤷🏻. And we can move with other tests.

Reply:
Noice! LGTM.